themes/bootstrap3/templates/RecordTab/holdingsils/extended.phtml

```diff
@@ -24,35 +25,26 @@
             <?=$this->transEsc('On Reserve - Ask at Circulation Desk')?><br>
           <?php endif; ?>
 
-          <?php if ($holding['use_unknown_message'] ?? false): ?>
-            <span class="text-muted"><?=$this->transEsc('status_unknown_message')?></span>
-          <?php else: ?>
-            <?php if ($holding['availability']): ?>
-              <?php /* Begin Available Items (Holds) */ ?>
-              <?php
-                if ((int)$holding['availability'] === \VuFind\ILS\Logic\ItemStatus::STATUS_UNCERTAIN) {
-                  $statusClassEsc = 'text-warning';
-                  $status = $holding['status'];
-                  $schemaAvailEsc = 'http://schema.org/LimitedAvailability';
-                } else {
-                  $statusClassEsc = 'text-success';
-                  $status = is_int($holding['availability']) && !empty($holding['status']) ? $holding['status'] : 'Available';
-                  $schemaAvailEsc = 'http://schema.org/InStock';
-                }
-              ?>
-              <span class="<?=$statusClassEsc?>"><?=$this->transEsc($status)?><link property="availability" href="<?=$schemaAvailEsc?>"></span>
-            <?php else: ?>
-              <?php /* Begin Unavailable Items (Recalls) */ ?>
-              <span class="text-danger"><?=$this->transEsc($holding['status'])?><link property="availability" href="http://schema.org/OutOfStock"></span>
-              <?php if ($holding['returnDate'] ?? false): ?>
-                <span class="small return-date"><?=$this->escapeHtml($holding['returnDate'])?></span>
-              <?php endif; ?>
-              <?php if ($holding['duedate'] ?? false): ?>
-                <span class="small due-date"><?=$this->transEsc('Due')?>: <?=$this->escapeHtml($holding['duedate'])?></span>
-              <?php endif; ?>
-              <?php if (($holding['requests_placed'] ?? 0) > 0): ?>
-                <span class="request-count"><?=$this->transEsc('Requests')?>: <?=$this->escapeHtml($holding['requests_placed'])?></span>
-              <?php endif; ?>
+          <?php
+            $statusClassEsc = $this->availabilityStatus()->getClass($availabilityStatus);
```

Comment:
Another instance of the question of whether or not we actually might need to escape the status classes... and of course this will also apply to standard.phtml below if changes are needed.

Reply:
See above.